app/src/main/java/com/mobiledevelopmentfoodapp/MainViewModel.kt

```diff
@@ -11,11 +11,11 @@ import kotlinx.coroutines.launch
 class MainViewModel : ViewModel() {
 
     var restaurant : MutableLiveData<List<Food>> = MutableLiveData<List<Food>>()
-    var RestaurantService : RestaurantService = RestaurantService()
+    var restaurantService : RestaurantService = RestaurantService()
```

Comment:
Changed this to adhere to proper camel case in variable names

Reply:
Good.